remoteappmanager/handlers/home_handler.py

```diff
@@ -254,7 +252,6 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         Container
         """
 
-        user_name = orm_user.name
```

Comment:
The `account` object is only needed by the database and no need for anything else, its implementation is defined by the database (orm or csv_db) and the object's state is inquired there.